agent.py

```diff
@@ -4,9 +4,13 @@
 from langchain.text_splitter import RecursiveCharacterTextSplitter
 from langchain.vectorstores import FAISS
 from langchain.llms import VertexAI
-
 from langchain.chains import ConversationalRetrievalChain
 
+from dotenv import load_dotenv

```

Comment:
Added a suggestion for the code in this PR.
```suggestion
from dotenv import load_dotenv # test```